.dev_scripts/benchmark_regression.py

```diff
@@ -55,251 +74,164 @@ def main():
     args = parse_args()
 
     if args.root_work_dir is None:
-        date = datetime.datetime.now()
-        args.root_work_dir = 'work_dirs/benchmark_regression_' + \
-                             f'{date.year}{date.month:02d}{date.day:02d}_' + \
-                             f'{date.hour:02d}{date.minute:02d}'
-    mmcv.mkdir_or_exist(os.path.abspath(args.root_work_dir))
+        # get the current time stamp
+        ts = time.ctime()
```

Comment:
Consider using `time.strftime` to directly get the time string?

Reply:
Got it.